common-arcana.lic

```diff
@@ -92,7 +92,7 @@ module DRCA
     when /^Your target pattern dissipates/, /^You can't cast that at yourself/, /^You need to specify a body part to consume/
       fput('release spell')
     when /^At the ritual's peak, your prophetic connection blooms a thousand-fold/
-      Flags.reset('rtr-expire')
+      Flags.add('rtr-expire', get_data('spells').spell_data['Read the Ripples']['expire'])
```

Comment:
I was having some issue with the flag being raised. I don't actually know what was causing it. The problem was the RtR expire flag would never be raised when the expire message came. It looks like adding it is the same as resetting it.